content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -4,211 +4,193 @@ Manage vendor bills
-(:dfn:`Reference Number`) can be changed from this pop-up. Once ready, click :guilabel:`Create
-Payment` to finish creating the vendor bill. Doing so causes a green **In Payment** banner to
-display on the :abbr:`RFQ (request for quotation)` form.
+(:dfn:`Reference Number`) can be edited from this pop-up window, if necessary.
 
-Create and manage vendor bills in Accounting
-============================================
+Once ready, click :guilabel:`Create Payment` to finish creating the :guilabel:`Vendor Bill`. Doing
+so displays a green :guilabel:`Paid` banner on the |RfQ| form.
+
+Manage vendor bills in Accounting
+=================================
 
 Vendor bills can also be created directly from the *Accounting* app, **without** having to create a
-purchase order first. To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`, and
-click :guilabel:`Create`. Doing so reveals a blank vendor bill detail form.
+purchase order first.
+
+Navigate to :menuselection:`Accounting app --> Vendors --> Bills`, and click :guilabel:`New`. Doing
+so reveals a blank vendor bill form.
+
+Add a vendor in the :guilabel:`Vendor` field. Then, in the :guilabel:`Invoice Lines` tab, click
+:guilabel:`Add a line` to add products.
+
+On the product line, select a product from the drop-down menu in the :guilabel:`Product` field, and
+enter the quantity to order in the :guilabel:`Quantity` field.
 
-On this blank vendor bill detail form, add a vendor in the :guilabel:`Vendor` field, and add
-products to the :guilabel:`Product` lines (under the :guilabel:`Invoice Lines` tab), by clicking
-:guilabel:`Add a line`. Then, add a bill date in the :guilabel:`Bill Date` field, and any other
-necessary information. Finally, click :guilabel:`Confirm` to confirm the bill.
+Select a :guilabel:`Bill Date`, and configure any other necessary information. Finally, click
+:guilabel:`Confirm` to confirm the bill.
 
-From here, click the :guilabel:`Journal Items` tab to view (or change) the :guilabel:`Account`
-journals that were populated based on the configuration on the corresponding :guilabel:`Vendor` and
+Once confirmed, click the :guilabel:`Journal Items` tab to view the :guilabel:`Account` journals.
+These journals are populated based on the configuration on the corresponding :guilabel:`Vendor` and
 :guilabel:`Product` forms.
 
-Then, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note` to add credit or debit notes
-to the bill. Or, add a :guilabel:`Bill Reference` number (while in :guilabel:`Edit` mode).
+If necessary, click :guilabel:`Credit Note` to add a credit note to the bill. Additionally, a
+:guilabel:`Bill Reference` number can be added.
 
-Then, when ready, click :menuselection:`Register Payment --> Create Payment` to complete the
-:guilabel:`Vendor Bill`.
+Once ready, click :guilabel:`Register Payment`, followed by :guilabel:`Create Payment`, to complete
+the :guilabel:`Vendor Bill`.
```

Comment:
This was not available, nor applicable, during my review. Once I clicked 'Confirm' a 'Paid' banner appeared and the status of the Vendor Bill changed from 'Draft' to 'Posted' -- perhaps an additional step is needed for this element to come into play? Or, perhaps, this portion simply is no longer accurate/needed for the process. Either way, it's worth double-checking during your revisions.

Reply:
In my database, it works as described. I could not find a setting that changes that.